operators/src/source/ogr_source/mod.rs

```diff
@@ -1034,6 +1092,7 @@ where
         chunk_byte_size: usize,
     ) -> Result<FeatureCollection<G>> {
         let was_spatial_filtered_by_ogr = feature_iterator.was_spatial_filtered_by_ogr();
+        let was_time_filtered_by_ogr = false; // TODO currently, we don't know whether a filter was applied at this point, so we filter again
```

Comment:
can you not pass this info into the function?

Reply:
https://github.com/geo-engine/geoengine/pull/961/commits/e8e7d0f78a8a0220399e9feb62763dc199a895a9

Reply:
https://github.com/geo-engine/geoengine/pull/961/commits/4dd828dcb5265edac96707e2d4fea3c8a2a66805